qemu/tests/cfg/hotplug_virtio_mem.cfg

```diff
@@ -0,0 +1,32 @@
+- hotplug_virtio_mem:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
+    no s390x
+    type = hotplug_virtio_mem
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    slots_mem = 20
+    maxmem_mem = 80G
+    mem = 4096
```

Comment:
Suggest `mem_fixed`  as the mem size for numa node is hard code, need ensure no other configuration overwrite it.

Reply:
Changed now, thanks!